cmd/webhook_v2.go

```diff
@@ -71,7 +71,7 @@ func newWebhookV2Cmd() *cobra.Command {
 			}
 
 			fmt.Println("Writing scaffold for you to edit...")
-			fmt.Println(filepath.Join("api", o.res.Version,
+			fmt.Println(filepath.Join(o.res.Group, "api", o.res.Version,
```

Comment:
Should this be `fmt.Println(filepath.Join("api", o.res.Group, o.res.Version, ...)`

Reply:
good catch. 🥇 done.